sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorExporterTraceExtensions.cs

```diff
@@ -72,7 +72,7 @@ public static class AzureMonitorExporterTraceExtensions
 
                 return new CompositeProcessor<Activity>(new BaseProcessor<Activity>[]
                 {
-                    new StandardMetricsExtractionProcessor(),
+                    new StandardMetricsExtractionProcessor(new AzureMonitorMetricExporter(exporterOptions)),
```

Comment:
Does this mean a entirely new metric exporter here?

Reply:
Yes, there isn't a way to use the one created by user (if they create one)

Reply:
This means with disto there are 4 exporter threads. Does statsbeat use its own thread?

Reply:
unfortunately yes. Does statsbeat use its own thread? Yes, so that will be 5.